runtime/src/main/scala/com/ing/baker/runtime/akka/actor/process_instance/ProcessInstanceRuntime.scala

```diff
@@ -148,7 +148,7 @@ trait ProcessInstanceRuntime[P, T, S, E] {
       else
         enabledParameters(instance.petriNet)(instance.availableMarking).get(transition) match {
           case None ⇒
-            (instance, Left(s"Not enough consumable tokens"))
+            (instance, Left(s"Not enough consumable tokens. This might have been caused because the event has already been fired and it is not configured to fire more than once, use withSensoryEventNoFiringLimit on the recipe interaction if such behaviour is desired"))
```

Comment:
Although this is a much better error message its not completely true.
It is also possible to configure the max firing limit to any number, for example 3.
The current error message does not take this into account.

Reply:
done! :D